libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -233,16 +233,16 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     explorerViews: [],
   },
-  avalanche: {
+  avalanche_c_chain: {
     type: "CryptoCurrency",
-    id: "avalanche",
-    coinType: 9000,
-    name: "Avalanche",
-    managerAppName: "Avalanche",
-    ticker: "AVAX",
-    scheme: "avalanche",
+    id: "avalanche_c_chain",
+    coinType: CoinType.ETH,
```

Comment:
Just to be sure we are going to use only the eth implementation of avax right ?

Reply:
There will be the P Chain as well which is not based on ETH

Reply:
So cointype set as ETH is good ? I don't really know but according to the list shouldn't it be 9006 or something like that ?